python/ct/crypto/verify.py

```diff
@@ -178,49 +191,135 @@ def _create_dst_entry(sct, chain):
 
     return entry
 
-class LogVerifier(object):
-    """CT log verifier."""
-    __ECDSA_READ_MARKERS = ("PUBLIC KEY", "ECDSA PUBLIC KEY")
-    __ECDSA_WRITE_MARKER = "ECDSA PUBLIC KEY"
+class RsaPublicKey(object):
```

Comment:
Extract to a separate file, so only the LogVerifier remains here and each of the RSA/ECDSA specific code lives in a separate module.
Tidier, each file would be shorter and with narrower scope.